integration-tests/docker/test_env/cl_node.go

```diff
@@ -35,7 +36,7 @@ import (
 type ClNode struct {
 	EnvComponent
 	API            *client.ChainlinkClient
-	NodeConfigOpts node.NodeConfigOpts
+	NodeConfigOpts node_config.NodeConfig
```

Comment:
Continuing from the above comment, you're double naming here. It'd make much more sense for the package to be just `node` and you reference the config as `node.Config`